### PR TITLE
Fix and re-enable UfsFileIn/OutStreamTest

### PR DESCRIPTION
**Pull request by voddle**
### What changes are proposed in this pull request?
  UfsFileIn/OutStream Test runnable now

### Why are the changes needed?
  Need UfsFileIn/OutStream Test

### Does this PR introduce any user facing changes?
  No

### CODE REVIEW

**Comment by YichuanSun** [comment]
Thanks! LGTM.

**Review by YichuanSun — APPROVED** [review]

File: dora/core/client/fs/src/test/java/alluxio/client/file/ufs/AbstractUfsStreamTest.java [hunk 1]
```diff
@@ -55,7 +55,7 @@ public abstract class AbstractUfsStreamTest {
 
   @Parameterized.Parameters
   public static Collection<Object[]> data() {
-    return Arrays.asList(new Object[][] {{false}, {true}});
```
Comment (jiacheliu3) [inline dora/core/client/fs/src/test/java/alluxio/client/file/ufs/AbstractUfsStreamTest.java:58]: why doesn't this work when `true`?

Comment (YichuanSun) [reply dora/core/client/fs/src/test/java/alluxio/client/file/ufs/AbstractUfsStreamTest.java:58]: please delete the Parameterized.

File: dora/core/client/fs/src/test/java/alluxio/client/file/ufs/AbstractUfsStreamTest.java [hunk 2]
```diff
@@ -78,9 +57,10 @@ public void before() {
     mRootUfs = new AlluxioURI(ufs);
     UnderFileSystemFactoryRegistry.register(new LocalUnderFileSystemFactory());
     final FileSystemOptions fileSystemOptions =
-        FileSystemOptions.Builder
-            .fromConf(mConf)
+        new FileSystemOptions.Builder()
             .setUfsFileSystemOptions(new UfsFileSystemOptions(ufs))
+            .setDoraCacheEnabled(false)
+            .setDataCacheEnabled(false)
```
Comment (jiacheliu3) [inline dora/core/client/fs/src/test/java/alluxio/client/file/ufs/AbstractUfsStreamTest.java:63]: fyi @dbw9580 @JiamingMai @LuQQiu  In the test for UfsFileStream we are manually disabling meta/data client cache. Let us know if this is incorrect. Thanks!

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks for the fix!

File: dora/core/client/fs/src/test/java/alluxio/client/file/ufs/AbstractUfsStreamTest.java (hunk 2 above)
Comment (dbw9580) [reply dora/core/client/fs/src/test/java/alluxio/client/file/ufs/AbstractUfsStreamTest.java:63]: looks good to me

Comment (YichuanSun) [inline dora/core/client/fs/src/test/java/alluxio/client/file/ufs/AbstractUfsStreamTest.java:63]: same line?

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**